build.gradle

```diff
@@ -79,7 +79,7 @@ afterEvaluate {
     // Create a task for each JUnit test class whose name is the same as the JUnit class name.
     sourceSets.test.allJava.filter { it.path.contains("${picoPath}/src/test/java") }.forEach { file ->
         String junitClassName = file.name.replaceAll(".java", "")
-        String testName = junitClassName.replaceAll("Test", "")
+        String testName = junitClassName.replaceAll("Tests", "")
```

Comment:
In the Checker Framework the test drivers all end with `Test`: https://github.com/typetools/checker-framework/tree/master/checker/src/test/java/tests
So we should do the renaming from xxxTests -> xxxTest.

The corresponding code in the CF build file https://github.com/typetools/checker-framework/blob/master/build.gradle#L533
doesn't substitute names. Why is this neede?

Reply:
I think this is not mandatory to substitute names, just for showing the simple names in terminal.

I will rename it.

Reply:
After substituting names, if we execute `./gradlew tasks`, we will get something like this:

```
Verification tasks
------------------
check - Runs all checks.
ImmutabilityInferenceTest - Run ImmutabilityInference tests.
ImmutabilityTypecheckTest - Run ImmutabilityTypecheck tests.
test - Runs the unit tests.
```

If we do not substitute names, it will be like:

```
Verification tasks
------------------
check - Runs all checks.
ImmutabilityInferenceTest - Run ImmutabilityInferenceTest tests.
ImmutabilityTypecheckTest - Run ImmutabilityTypecheckTest tests.
test - Runs the unit tests.
```

So I think maybe it is better to substitute names to get a better description.